Core/Src/monitor.c

```diff
@@ -139,6 +147,71 @@ void vPedalsMonitor(void* pv_params)
 		// MAX_ADC_VAL_12B) && 	!is_timer_active(&diff_timer)) 	start_timer(&diff_timer,
 		// PEDAL_FAULT_TIME); else 	cancel_timer(&diff_timer);
 
+
+		/* Evaluate accelerator faults */
```

Comment:
Can you add in more comments in this section to explain what is actually happening? Feels like a very large chunk of logic that is a bit hard to decipher.

Reply:
Added necessary comments. If needed, will make it more comprehensive.